pkg/virtual/framework/dynamic/apiserver/handler_test.go

```diff
@@ -191,11 +244,27 @@ func TestRouting(t *testing.T) {
 			ExpectDelegateCalled: false,
 			ExpectStatus:         200,
 			ExpectResponse: func(t *testing.T, r *http.Response, b []byte) {
-				if r.Header.Get("Content-Type") == "application/json" && r.StatusCode == 200 {
-					require.Equal(t,
-						"{\"kind\":\"APIGroup\",\"apiVersion\":\"v1\",\"name\":\"custom\",\"versions\":[{\"groupVersion\":\"custom/v1\",\"version\":\"v1\"}],\"preferredVersion\":{\"groupVersion\":\"custom/v1\",\"version\":\"v1\"}}\n",
```

Comment:
I don't think that asserting on some string here is correct - JSON has no ordering w.r.t. fields at the same level of nesting. anyway. Plus, actually comparing objects gives us sensible diffs.